src/components/EventCardList/EventCardList.jsx

```diff
@@ -83,17 +85,19 @@ const EventCardList = ({eventList}) => {
 	return (
 		<EventListContainer data-testid="list">
 			{error && <div>An error occured: {error}</div>}
-			{data.map(singleEvent => {
-				return (
-					<EventCard
-						data-testid="listItem"
-						key={singleEvent._id}
-						event={singleEvent}
-						deleteEvent={deleteEvent}
-						bookmark={bookmarkEvent}
-					></EventCard>
-				);
-			})}
+			{data
+				.filter(event => event.categories.includes(active))
+				.map(singleEvent => {
```

Comment:
You don't need to call the single element single Event, just call it event.

Reply:
Done